.gitignore

```diff
@@ -17,6 +17,7 @@ TAGS
 
 # ignore generated files.
 cmd/influxd/version.go
+cmd/influxd/term
```

Comment:
@otoolep In hindsight, that file shouldn't be gitignored. I believe it was leftover from a test that didn't clean up correctly for me. I will remove this commit.